docs/sesami-sdk/ratelimit.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 2
+---
+
+# Sesami Storefront API rate limit
+
+To ensure our platform remains stable and fair for everyone, Sesami storefront API, powering our Sesami storefront SDK is rate-limited. We use a request-based limiting strategy to enforce rate limits. We ask developers to use industry standard techniques for limiting calls, caching results, and re-trying requests responsibly.
+
+This is the current setup in 3 different buckets:
+
+- 10m:200
+- 10s:40
+- 1s:20
+
+Meaning that a user can't make more than 20 requests per second, more than 40 requests per 10 seconds and more than 200 in 10 minutes.
+
```

Comment:
This shouldn't be exposed to outside, clients should rely on `retry-after` header retuned on a `429` response